packages/block-editor/src/components/list-view/block.js

```diff
@@ -134,7 +134,7 @@ function ListViewBlock( {
 	const { isTreeGridMounted, expand, collapse } = useListViewContext();
 
 	const hasSiblings = siblingBlockCount > 0;
-	const hasRenderedMovers = showBlockMovers && hasSiblings;
+	const hasRenderedMovers = true && hasSiblings;
```

Comment:
I didn't intend to commit this line, but we can keep it for now to make it easier to review the block mover button in the list view. I will remove this line before merging